stories/form/form-date-input/form-date-input.stories.tsx

```diff
@@ -22,6 +22,10 @@ export const Default: StoryObj<Component> = {
         return (
             <StoryContainer>
                 <Form.DateInput label="This is the default date input" />
+                <Form.DateInput
+                    label="This is hideKeyboard set to true"
+                    hideKeyboard={true}
```

Comment:
for boolean values `that are true`, no need `={true}`

Reply:
Updated